Sources/SwiftInspectorVisitors/Tests/NestableTypeVisitorSpec.swift

```diff
@@ -929,6 +930,8 @@ final class NestableTypeVisitorSpec: QuickSpec {
             $0.name == "FooEnum"
               && $0.inheritsFromTypes.map { $0.asSource } == []
               && $0.parentType?.asSource == nil
+              && $0.properties.isEmpty
+              && $0.functionDeclarations.isEmpty
```

Comment:
this check fails on `main` today

Reply:
We don't need to do this today, but when this test fails it's gonna be really annoying to debug. It will tell us "expected matching.count of 1, got XYZ.

That forces us to debug the test, which defeats the purpose of having a robust test suite.

I'd suggest refactoring these tests so when they fail they're trivial to understand why they failed, but we don't need to that in this PR.